### ISSUE TITLE
Tip overlapped search box

### ISSUE THREAD

**Issue by ChristineWk** [joomla-extensions/patchtester#243]
#### Steps to reproduce the issue
see screenshot

#### System information (as much as possible)
see screenshot
#### Additional comments
![tool-tip-problem](https://user-images.githubusercontent.com/15275917/67148118-e78ad980-f29b-11e9-98cd-30b32db285ab.png)

**Comment by uglyeoin** [comment]
```
:focus + [role="tooltip"], :hover + [role="tooltip"] {
    display: block;
    position: absolute;
    bottom: 0;
}
```
This can be fixed by adding `bottom: 0;` to `/administrator/templates/atum/css/template.min.css?f6da6cc9d583349709a7026265890f4d`

I'm not sure if this is a pull request I can do.

**Comment by ChristineWk** [comment]
@uglyeoin 

Thank you for your investigation. Found via inspector required code, tried/added temporary  the bottom: 0; accordingly - but no success. Think, the reason might be that it's template.min.css compiled css. Maybe I should try directly in that CSS?

Also I don't know whether it affect other tool-tips.

**Comment by uglyeoin** [comment]
You added it to this rule?

`:focus + [role="tooltip"], :hover + [role="tooltip"] {`

It shouldn't matter if it is the compiled file as you are using the browser so it is not reloading it at that point.  If it isn't working perhaps you could add `!important` just to make sure your rule is taking precedent.  It worked for me though, see my screenshot for how I achieved it.

**Comment by richard67** [comment]
Please test #246 .

**Comment by richard67** [comment]
I've just tested. #246 is correct and fixes this issue. S I think this issue could be closed.

**Comment by ChristineWk** [comment]
@richard67 
OK I wanted allready, but don't know how :-)
Do I hv to load e.g. this: https://github.com/mangeshmane/patchtester/blob/master/administrator/components/com_patchtester/PatchTester/View/Pulls/tmpl/default.php

**Comment by richard67** [comment]
I have installed patchtester Patch Tester 4.0.0 Beta 2, then I have applied the changes from PR's #241 and #242 "manually", i.e. by editing the php files changed by these PR's. Then I've verified that I have this issue. Then I've applied the change in #246 , also "manually", and verified the issue has gone.

**Comment by richard67** [comment]
In addition I have verified that in other list views of the cms, the same scheme for the tool tip div's ID is used as it is after the correction made by PR #246. A lot easier it is just to apply the change done by PR #246 in the developer tools of your browser by using "Examine element" ("Element untersuchen" in German FireFox) and then just editing the id like that PR changes it and then check the result.

**Comment by ChristineWk** [comment]
Tried both. Changed accordingly: https://github.com/joomla-extensions/patchtester/pull/246/commits/eb5f87b2f0b3974352414086b48c20ada70cbf92
Via Developer tools FF (no success) and also directly via FTP above File. Hover OK, but not when clicking in box.

**Comment by richard67** [comment]
Strange. Anyway, it is correct.

Trust me (said the Transsilvanian cab driver to the scared British toruist on the way to that spooky castle ;-) ).

**Comment by ChristineWk** [comment]
Of course, I trust you :-) Forever and ever (it's a song)

**Comment by uglyeoin** [comment]
I'm confused, it says Hackwar has merged the other two pulls.  Does that mean they are now closed?  So long as we use the latest version of Patchtester we don't need to do that part just test the final part?

**Comment by richard67** [comment]
PR's #241 and #242 have been merged, but no new beta release has been built after that, and so if using the latest beta "4.0.0 Beta 2" it still needs to apply those PR's, too.

**Comment by richard67** [comment]
P.S.: For testing the PR #246 for this issue you don‘t need the other 2 PR‘s, you need them only if you want to use patchtester to apply and revert PR‘s.

**Comment by ChristineWk** [comment]
I'm also confused. (via FTP manually) Days before I hv changed accordingly: https://github.com/mangeshmane/patchtester/commit/eb5f87b2f0b3974352414086b48c20ada70cbf92

Just now in addition: https://github.com/joomla-extensions/patchtester/pull/241/commits/380831c017b7d3af52438f93f98f40c4065f1877 & https://github.com/joomla-extensions/patchtester/pull/242/commits/63d9019052173d3a0126ab49d8ddaac23a8511a3

**Comment by richard67** [comment]
@ChristineWk Ah I just see you wrote hover is ok, only clicking in the box is not? I have to test again tonight after work... maybe I‘ve tested only hovering.

**Comment by richard67** [comment]
@ChristineWk I've just tested again, it works for me with hover and also when cursor is in the search box. See screen shot.
![patchtester-search-box-tool-tip](https://user-images.githubusercontent.com/7413183/68610684-9e076600-04b8-11ea-803c-ee2e549d06e8.png)

**Comment by richard67** [comment]
Maybe the difference is that I test on current 4.0-dev branch of the CMS and not any older alpha or nightly build?

**Comment by richard67** [comment]
Here is a link to a patched install package for the patchtester which contains all recently merged changed plus the change from PR #246 : [https://test5.richard-fath.de/com_patchtester_4.0.0.beta3-dev-3.zip](https://test5.richard-fath.de/com_patchtester_4.0.0.beta3-dev-3.zip)

Install this on a clean 4.0dev of the CMS or a nightly build of last night. Clean means not updated from any alpha because this is not supported yet, and also don't update the patchtester but make a new installation of it.

**Comment by ChristineWk** [comment]
@richard67 
What I'm currently use is this: 

Joomla! Patch Tester | Administrator | Component | 4.0.0-beta2 | 17-October-2019
Site | File | No Problems | 4.0.0-2019-10-06 | ‎4.0.0-beta1-dev

So, I will delete the space and install everything new ..... But it will takes time :-)
Thanks for help.

**Comment by ChristineWk** [comment]
So, completely new full package nightly build of last night.
Also installed your .../com_patchtester_4.0.0.beta3-dev-3

System shows:

Site | File | No Problems | 4.0.0-2019-10-17 | ‎4.0.0-beta1-dev
Administrator | Component | No Problems | 4.0.0 | ‎4.0.0-beta**2**

I'm proudly present :-)

![patchtester_beta3](https://user-images.githubusercontent.com/15275917/68623394-aff70200-04d4-11ea-9ccb-043b0da181bb.png)

**Comment by richard67** [comment]
So all ok? If so, you can close this issue.

**Comment by richard67** [comment]
And if ok, could you also comment in PR #246 that you have tested with success?

**Comment by ChristineWk** [comment]
yes OK, will check PR #246.

**Comment by richard67** [comment]
PR #246 was included in my zip so you have tested it already.

**Comment by richard67** [comment]
@ChristineWk Finally you can close this issue. There should be a button for this on GitHub for you in issues which you have created. Best is closing with a comment "solved with PR #246".

**Comment by ChristineWk** [comment]
Solved with PR #246

**Comment by richard67** [comment]
Perfect.
